src/clj/ataru/person_service/person_integration.clj

```diff
@@ -36,7 +36,9 @@
       (assoc basic-fields :birthDate (extract-birth-date application)))))
 
 (defn upsert-and-log-person [person-service application-id]
-  (let [person-to-send (extract-person (application-store/get-application application-id))]
+  (let [person-to-send (-> (application-store/get-application application-id)
+                           (dissoc :secret)
```

Comment:
Is (dissoc :secret) really necessary? extract-person doesn't include it anyway in the result.